site/content/docs/essentials.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
General approach question: do we have one docs page per significant feature of merge all into one here. At this point I'm not sure it matters that much. My gut instinct is one page per feature is a bit better.